### DISCUSSION TITLE
can't change revolt username, spits out error

### DISCUSSION THREAD

**Discussion by therealonlygithub1** [stoatchat/discussions#446] [Q&A] [answered]
every time I try to change my revolt username it spits out an error even though my password is correct. Anyone got any solutions

**Comment by Rexogamer** [comment, marked as answer]
What are you trying to change it to? Bear in mind that usernames are currently unique.

**Reply by therealonlygithub1** [reply]
makes sense for the error, it should display a message saying username is taken until the #0000 are added to revolt. So other users don't get confused
